docs/development.md

```diff
@@ -307,23 +307,38 @@ Run the following command to build the deployment config `hack/install.yaml`
 that includes all the necessary kubernetes resources:
 
 ```bash
-kubebuilder create config \
-    --controller-image "<containerregistry>/<username>/federation-v2:test" \
-    --name federation
+INSTALL_YAML="hack/install.yaml"
+IMAGE_NAME="<containerregistry>/<username>/federation-v2:test"
+INSTALL_YAML="${INSTALL_YAML}" IMAGE_NAME="${IMAGE_NAME}" scripts/generate-install-yaml.sh
 ```
 
-Once the installation YAML config `hack/install.yaml` is created, we need to
-update it to modify a few fields.
+Once the installation YAML config `hack/install.yaml` is created, you are able
```

Comment:
Nit: My english is not that good, so I might be wrong also, but `you would be able to apply....` in place of `you are able to apply.. ` sounds more right somehow.

Reply:
@irfanurrehman Either works, but using *would* implies future tense, whereas it's written up in the present tense as if the user is currently performing the steps.